### PR TITLE
feat-fe: bootstrap frontend app

### LINKED ISSUE #19: feat-fe: bootstrap frontend

### PR DESCRIPTION
**Pull request by iruszpel**
Closes #19

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-is": "18.2.0",
-    "styled-components": "5.3.6",
+    "react95": "^4.0.0",
```
Comment (iruszpel) [inline frontend/package.json:16]: 👍

Comment (sobczal2) [reply frontend/package.json:16]: 👍

**Review by sobczal2 — DISMISSED** [review]
lgtm

**Comment by legeof008** [comment]
@iruszpel @sobczal2 Is this pr still necessary ?  `feature/fe/payments` seems to cover most of it.

**Pull request closed without merging**